internal/core/regularsurface.cpp

```diff
@@ -55,6 +64,10 @@ AffineTransformation AffineTransformation::inverse_from_rotation(
     }}));
 }
 
+bool Plane::operator==(const Plane& other) const {
+    return this->m_transformation == other.m_transformation;
```

Comment:
Ideally you should compare the inverse here too

Reply:
Why?

Reply:
hmm ok, its fine in this case as both transformation are made from the same parameters.